client/client.go

```diff
@@ -281,21 +271,93 @@ func newHttpClient() *http.Client {
 	return &http.Client{}
 }
 
+func loadServerTrust(ctx *openssl.Ctx, conf *Config) (*openssl.Ctx, error) {
+	// Trust CA's in the standard location (/etc/ssl/certs), and the
```

Comment:
Also, @merlin-northern, should we possibly load the certificates explicitly here, instead of passively through the `LoadVerifyLocations` call? So that it is similar to `loadClientTrust`? It would leave us with more control I think. At least after you've finished the `SSL_DEFAULT_LOCATION` implementation?

Reply:
I think we should use LoadVerifyLocations as it calls SSL_CTX_load_verify_locations which is the proper way to load the trust store. I think it can be like that. As soon as we merge this one, then we can move with [pull/587](https://github.com/mendersoftware/mender/pull/587) and then the only modification here, will be: add `SetDefaultVerifyLocations` call.
@oleorhagen do you need to hard code "/etc/ssl/certs/" here for the sake of the tests, or for some other reason? if not, I would drop it, since technically it can break in case the /etc/ssl does not exist. (I am not sure how SSL_CTX_load_verify_locations behaves when the directory does not exist). keep it in if it is needed for this PR.

Reply:
`/etc/ssl/certs` is only a temporary fix, until #587 and your confirmation on it :)

Reply:
See https://github.com/mendersoftware/mender/pull/588#discussion_r470489503

Reply:
@oleorhagen since https://github.com/mendersoftware/openssl/pull/11 and https://github.com/mendersoftware/mender/pull/587 are in place, maybe lets just add `ctx.SetDefaultVerifyLocations()` here, after rebase? (just a suggestion I can do it afterwards)

Reply:
Added